api/HTMLElement.json

```diff
@@ -2356,7 +2356,7 @@
               "version_added": null
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
I'm removing this entry in https://github.com/mdn/browser-compat-data/pull/7484, `onModule` isn't a thing.